srml/authorship/src/lib.rs

```diff
@@ -101,16 +156,16 @@ pub trait FilterUncle<Header, Author> {
 
 	/// Do additional filtering on a seal-checked uncle block, with the accumulated
 	/// filter.
-	fn filter_uncle(header: &Header, acc: Self::Accumulator)
-		-> Result<(Option<Author>, Self::Accumulator), &'static str>;
+	fn filter_uncle(header: &Header, acc: &mut Self::Accumulator)
```

Comment:
Changed `acc` to be a mutable reference because we also need it in case of error so that filtering might be able to skip invalid uncle and continue. Returning `acc` for both `Ok` and `Err` results is way less readable and more verbose code.